app/models/manageiq/providers/amazon/cloud_manager/refresh_parser.rb

```diff
@@ -24,19 +24,17 @@ def ems_inv_to_hashes
     log_header = "MIQ(#{self.class.name}.#{__method__}) Collecting data for EMS name: [#{@ems.name}] id: [#{@ems.id}]"
 
     $aws_log.info("#{log_header}...")
-    AWS.memoize do
```

Comment:
Memoization has been removed from V2
I have posted for clarification to the new usage model here:

https://ruby.awsblog.com/post/Tx2I4U7QAOJYNDY/Ruby-SDK-Version-2-and-Memoization